package.json

```diff
@@ -66,9 +66,11 @@
     "src"
   ],
   "publishConfig": {
-    "registry": "https://npm.pkg.github.com/"
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
   },
   "contributors": [
-    "Neilzon Jobert Calosa <neilzon.calosa@getorbital.com> (https://github.com/NeilzonC)"
+    "Neilzon Jobert Calosa <neilzon.calosa@getorbital.com> (https://github.com/NeilzonC)",
+    "Leonardo Louie Ordonez <leonardo.ordonez@getorbital.com> (https://github.com/LOpayperform)"
   ]
```

Comment:
I don't believe we should have this section.

Reply:
yeah I am gonna remove it :D